cmd/sshocker/run.go

```diff
@@ -42,6 +42,11 @@ var (
 			Usage: "enable sshfs nonempty",
 			Value: false,
 		},
+		&cli.StringFlag{
+			Name:  "sshfs-idmap",
+			Usage: "set sshfs idmap option",
+			Value: "none",
+		},
```

Comment:
I'm feeling we should rather add generic `-o` to support arbitrary mount options.

Addition of `sshfs-nonempty` was design mistake and should be deprecated in favor of `-o nonempty`